buổi-9/bai1/bai-1.js

```diff
@@ -0,0 +1,26 @@
+function tinhLuong(){
+    var  luong = window.prompt('Nhập vào lương của bạn: ')
```

Comment:
Không được dùng `var`
```suggestion
    const  salary = window.prompt('Nhập vào lương của bạn: ')
```